index.js

```diff
@@ -1,27 +1,27 @@
-const {Cc,Ci,Cr} = require("chrome");
-
-var self = require('sdk/self');
-var pageMod = require("sdk/page-mod");
-var buttons = require('sdk/ui/button/action');
-var tabs = require("sdk/tabs");
-
-// TODO: Make these configurable in prefs?
-var CONTENT_URL = 'http://localhost:8899/stub-content/index.html';
-var ALLOWED_ORIGINS = [
-  'http://localhost:8899/*',
-  'https://ideatown.firefox.com/*',
-  'https://mozilla.github.io/idea-town-addon/stub-content/',
-  'https://lmorchard.github.io/idea-town-addon/stub-content/'
-];
-
-pageMod.PageMod({
-  include: ALLOWED_ORIGINS,
+/*
+ * This Source Code is subject to the terms of the Mozilla Public License
+ * version 2.0 (the "License"). You can obtain a copy of the License at
+ * http://mozilla.org/MPL/2.0/.
+ */
+
+const {Cu} = require("chrome");
+const self = require('sdk/self');
+const tabs = require("sdk/tabs");
+const pageMod = require("sdk/page-mod");
+const buttons = require('sdk/ui/button/action');
+const cfg = require("./package.json").CONFIG;
+
+var mod = pageMod.PageMod({
```

Comment:
`var` snuck in here

Might not hurt to hang onto eslint (whether via gulpfile or npm script), something we can address in a follow-on PR.